llvm/lib/Support/PrefixMapper.cpp

```diff
@@ -278,6 +222,15 @@ Error TreePathPrefixMapper::add(const MappedPrefix &Mapping) {
   StringRef New = Mapping.New;
   if (Error E = canonicalizePrefix(Old))
     return E;
-  PM.add(MappedPrefix{Old, New});
-  return Error::success();
+  return PrefixMapper::add(MappedPrefix{Old, New});
+}
+
+StringRef
+TreePathPrefixMapper::mapDirEntry(const vfs::CachedDirectoryEntry &Entry,
+                                  StringSaver &Saver) {
+  StringRef TreePath = Entry.getTreePath();
+  SmallString<256> PathBuf;
+  Optional<StringRef> Mapped =
+      cantFail(PrefixMapper::mapImpl(TreePath, PathBuf));
```

Comment:
Why is this one `cantFail` but the call to `PrefixMapper::mapImpl`  in `TreePathPrefixMapper::mapImpl` is not?

Reply:
I made both `cantFail` for consistency.